# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cache/DBSchema.cpp
# Commit: ddf4012a7652
# Full Hash: ddf4012a7652e36d144fc43bc99335276deeafbe
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2019-05-22 03:40:58
# Regressor Bug: 965637
# File Overlap Count: 1
# Description:
#   Bug 965637: Move CSP from Principal into Client, part 1: backend changes. r=mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27654
# ==============================================================================

diff -r 6c2047bc2f3e -r ddf4012a7652 dom/cache/DBSchema.cpp
--- a/dom/cache/DBSchema.cpp	Tue May 21 22:51:54 2019 +0000
+++ b/dom/cache/DBSchema.cpp	Tue May 21 23:14:27 2019 +0000
@@ -2492,9 +2492,6 @@
     nsCString origin;
     url->Origin(origin);
 
-    // CSP is recovered from the headers, no need to initialise it here.
-    nsTArray<mozilla::ipc::ContentSecurityPolicy> policies;
-
     nsCString baseDomain;
     rv = url->BaseDomain(baseDomain);
     if (NS_WARN_IF(NS_FAILED(rv))) {
@@ -2503,8 +2500,7 @@
 
     aSavedResponseOut->mValue.principalInfo() =
         Some(mozilla::ipc::ContentPrincipalInfo(attrs, origin, specNoSuffix,
-                                                Nothing(), std::move(policies),
-                                                baseDomain));
+                                                Nothing(), baseDomain));
   }
 
   bool nullPadding = false;